security/snort/Makefile

```diff
@@ -44,7 +45,7 @@ OPTIONS_GROUP=		ADDONS DEV
 OPTIONS_GROUP_ADDONS=	BARNYARD PULLEDPORK
 OPTIONS_GROUP_DEV=	DBGSNORT
 OPTIONS_SUB=		yes
-OPTIONS_DEFAULT=	GRE NORMALIZER BARNYARD \
+OPTIONS_DEFAULT=	GRE NORMALIZER TARGETBASED BARNYARD \
```

Comment:
Instead of changing OPTIONS_DEFAULT, creating an extra diff from upstream, it's better just to set this option on make.conf used by poudriere to build packages at https://github.com/pfsense/pfsense/blob/master/tools/conf/pfPorts/make.conf#L147